.github/workflows/build_and_test.yml

```diff
@@ -39,3 +39,49 @@ jobs:
 
       - name: Run unit tests
         run: yarn test-unit
+
+  build:
+    runs-on: ${{ matrix.os }}
+    name: Build on ${{ matrix.os }}
+    needs: lint-test
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+          registry-url: 'https://registry.npmjs.org'
+          cache: 'yarn'
+
+      - name: Setup Yarn
+        run: |
+          npm uninstall -g yarn
+          npm i -g yarn@1.22.10
+          yarn install --frozen-lockfile
+
+      - name: Path filter
+        uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          filters: |
+            src:
+              - 'src/**'
+              - package.json
+              - yarn.lock
+            docs:
+              - 'src-docs/**'
+              - package.json
+              - yarn.lock
```

Comment:
To reviewers: should other related build files be added here? Ex. `postcss.config.js`, `tsconfig.json`, ...

Reply:
yeah, I think adding those is a good idea. It's hard to know whether an allowlist or a denylist is easier in this case.